.github/workflows/ruby-build.yml

```diff
@@ -98,6 +98,7 @@ jobs:
           key: ruby-build
       - name: Build Query Pack
         run: |
+          rm -rf target/packs
```

Comment:
Looks good to me, but how did this folder end up in the cache?

Reply:
Because there was a `.cache` folder deep inside.  
That `.cache` folder is actually the one we want to restore, but if it had stayed in the previous location (the `0.4.4-dev` folder), then we wouldn't be able to use it.  
So that's another reason to use the `--compilation-dir` option.  

(The cache was saved when the Ruby analysis was version `0.4.4-dev`, and restored on version `0.4.5-dev`).